src/style.css

```diff
@@ -1 +1,217 @@
-/* empty source */
+* {
+  margin: 0;
+}
+
+.card {
+  width: 198px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.image {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+}
+
+.image__monitor {
+  padding: 0;
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-bottom: 0;
+}
+
+.title {
+  padding: 0;
+  margin: 0;
+  margin-top: 40px;
+  margin-right: 16px;
+  margin-left: 16px;
```

Comment:
do not use margins, use paddings for `card` instead
https://blog.hubspot.com/website/css-margin-vs-padding